openpype/modules/deadline/plugins/publish/collect_default_deadline_server.py

```diff
@@ -23,3 +34,16 @@ def process(self, context):
         context.data["defaultDeadline"] = deadline_module.deadline_urls["default"]  # noqa: E501
 
         context.data["deadlinePassMongoUrl"] = self.pass_mongo_url
+
+        deadline_servers = (context.data
+                            ["project_settings"]
+                            ["deadline"]
+                            ["deadline_servers"])
+        if deadline_servers:
+            deadline_server_name = deadline_servers[0]
+            deadline_webservice = deadline_module.deadline_urls.get(
+                deadline_server_name)
+            if deadline_webservice:
+                context.data["defaultDeadline"] = deadline_webservice
+                self.log.debug("Overriding from project settings with {}".format(  # noqa: E501
+                    deadline_webservice))
```

Comment:
In the future please try to make the log clear as to what it's actually doing, like including _what_ is being overridden from project settings since visually in the logs it might not be directly clear that the log is related to this particular plug-in.

```python
self.log.debug("Overriding default deadline webservice from project settings to: {}".format(deadline_webservice))
```

Let's try and avoid things like logging just "Changing to: x" or "Updating with: y" as from reading the logs it's not instantly clear what is being altered.